example-model/src/main/java/com/appunite/rx/example/model/dao/PostsDao.java

```diff
@@ -78,15 +90,37 @@ public PostsDao(@Nonnull final Scheduler networkScheduler,
                     }
                 });
 
-        posts = loadMoreSubject.startWith((Object) null)
-                .lift(mergePostsNextToken)
-                .compose(CacheSubject.behaviorRefCount(cacheProvider.<PostsResponse>getCacheCreatorForKey("posts", PostsResponse.class)))
-                .compose(ResponseOrError.<PostsResponse>toResponseOrErrorObservable())
-                .compose(MoreOperators.<PostsResponse>repeatOnError(networkScheduler))
-                .compose(MoreOperators.<ResponseOrError<PostsResponse>>refresh(refreshSubject))
-                .subscribeOn(networkScheduler)
+        Observable.combineLatest(
+                nameSubject,
+                bodySubject,
+                new Func2<String, String, AddPost>() {
+                    @Override
+                    public AddPost call(String name, String body) {
+                        return new AddPost(name, body);
+                    }
+                })
+                .lift(OperatorSampleWithLastWithObservable.<AddPost>create(sendStatus))
+                .flatMap(new Func1<AddPost, Observable<Response>>() {
+                    @Override
+                    public Observable<Response> call(AddPost post) {
+                        return guestbookService.createPost(post).subscribeOn(networkScheduler);
+                    }
+                })
+                .compose(ResponseOrError.<Response>toResponseOrErrorObservable())
+                .compose(MoreOperators.<Response>repeatOnError(networkScheduler))
```

Comment:
You shouldn't repeat posting after fail because you can accidently put multiple posts.
You can use use this only in "GET" methods of http